test/v1/services/NrsServiceSpec.scala

```diff
@@ -32,6 +31,8 @@ import v1.services.mocks.MockNrsConnector
 import java.nio.charset.StandardCharsets.UTF_8
 import java.util.UUID
 import scala.concurrent.Future
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AsyncWordSpec
 
 class NrsServiceSpec extends AsyncWordSpec with MockNrsConnector with Matchers {
```

Comment:
This is AsyncWordSpec rather than AnyWordSpec like all the others, is this deliberate?

Reply:
Yeah it was already AsyncWordSpec, this just changed where it was getting imported from